cmd/build-cross-compile/build-cross-compile.go

```diff
@@ -33,7 +19,6 @@ func main() {
 		buildName string
 		output    string
 		goBin     string
-		race      bool
```

Comment:
is the idea here that we don't need to build with `-race` to be able to test with `-race`?

Reply:
1. `go build` with race does the same as what race detector goes when you run `go test` - it prints a loud warning when your program races. https://go.dev/blog/race-detector
2. We've never built with race detection enabled.

Reply:
1. so go build will actually run the program to detect races? Or the resultant executable exits when it sees a race?
2. weren't we calling go build with `-race` in the case where `RACE_DETECTOR` was set before this change?